tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace BoShurik\TelegramBotBundle\Tests\DependencyInjection;
+
+use BoShurik\TelegramBotBundle\DependencyInjection\Configuration;
+use Matthias\SymfonyConfigTest\PhpUnit\ConfigurationTestCaseTrait;
+use PHPUnit\Framework\TestCase;
+
+class ConfigurationTest extends TestCase
+{
+    use ConfigurationTestCaseTrait;
+
+    protected function getConfiguration(): Configuration
+    {
+        return new Configuration();
+    }
+
+    public function testMinimalConfiguration(): void
+    {
+        $this->assertConfigurationIsValid(
+            [
+                [] // no values at all
```

Comment:
I think there is a configuration bug!
At least `api.token` should be provided.
Let's fit the configuration builder in another PR.